test/runtests.jl

```diff
@@ -1,5 +1,5 @@
 using Mocking
-Mocking.enable(; force=true)
+Mocking.activate()
```

Comment:
FYI with Mocking 0.7 you can put `Mocking.activate()` anywhere before you use a patch. Usually I place this after all the package imports